recipe/meta.yaml

```diff
@@ -12,25 +12,22 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  skip: false
-  number: 0
+  number: 1
 
 requirements:
   build:
     - python
     - setuptools
     - toolchain
     - future
-    - numpy x.x
+    - numpy 1.11
     - fftw
-    - openmp
   run:
     - python
     - setuptools
     - future
-    - numpy x.x
+    - numpy >=1.11
     - fftw
-    - openmp
```

Comment:
OSX and Linux compile C routines using openmp. But, you might be right, compilation may work without (if this is a standard C library included with gcc?)

Reply:
openmp conda package is the llvm one. You may want to add libgcc to run requirements for gcc one

Reply:
Circle seems happy without it (and the tests that run the openMP version pass), so it looks good. Thanks.

Reply:
Unfortunately it looks like OSX builds need the openmp package (as you might expect with it being LLVM).  Is there a way to just include it for those builds?